pkg/pushsync/metrics.go

```diff
@@ -14,48 +14,138 @@ type metrics struct {
 	// to be able to return them by Metrics()
 	// using reflection
 
-	SendChunkCounter      prometheus.Counter
-	SendChunkTimer        prometheus.Counter
-	SendChunkErrorCounter prometheus.Counter
-	MarkAndSweepTimer     prometheus.Counter
+	TotalChunksToBeSentCounter prometheus.Counter
+	TotalChunksSynced          prometheus.Counter
+	TotalChunksStoredInDB       prometheus.Counter
```

Comment:
This file is not gofmt-ed. An unexpected space character.

Reply:
@janos  My IDE was not not configured all these days for auto formatting. That was causing the issue. I fixed it and now this formatting issues should not happen.